src/app/stacks/multivisionstack.cpp

```diff
@@ -27,6 +27,11 @@ MultiVisionStack::MultiVisionStack(string _name, RenderOptions * _opts) {
 }
 
 MultiVisionStack::~MultiVisionStack() {
+  for (int i=0;i<threads.size();i++) {
+	threads.at(i)->stop();
```

Comment:
Sorry, I meant MultiVisionStack::stop(), which should take care of shutting down all the threads.